CHANGELOG.md

```diff
@@ -1,3 +1,96 @@
+### citus v10.2.0 (September 11, 2021) ###
+
+* Adds pg14 support
+
+* Acquires AccessShareLock before updating table statistics to verify the table
+  exists
+
+* Adds a parameter to cleanup metadata
+
+* Adds basic index support for columnar tables
+
+* Adds hash & btree index support for columnar tables
+
+* Adds helper UDFs for easy time partition management:
+  `get_missing_time_partition_ranges`,
+  `create_time_partitions`,
+  `drop_old_time_partitions`
+
+* Adds propagation of ALTER SEQUENCE
+
+* Adds support for ALTER INDEX ATTACH PARTITION
+
+* Adds support for CREATE INDEX ON ONLY
+
+* Allows more graceful failovers when replication factor > 1
+
+* Enables chunk group filtering to work with Params for columnar tables
+
+* Enables qual push down for joins including columnar tables
+
+* Enables transferring of data using binary encoding by default on PG14
+
+* Fixes a bug that allowed moving of shards belonging to a reference table
+
+* Fixes a bug about int and smallint sequences on MX
+
+* Fixes a bug that cause partitions to have wrong distribution key after
+  DROP COLUMN
+
+* Fixes a bug that caused `worker_append_table_to_shard` to write as superuser
+
+* Fixes a bug that caused `worker_create_or_alter_role` to crash with NULL input
+
+* Fixes a bug that causes pruning incorrect shard of a range distributed table
+
+* Fixes a bug that may cause crash while aborting transaction
+
+* Fixes a bug that prevents attaching partitions when colocated foreign key
+  exists
+
+* Fixes a bug where foreign key to reference table was disallowed
+
+* Fixes a bug with local cached plans on tables with dropped columns
+
+* Fixes a bug with `nextval('seq_name'::text)`
+
+* Fixes a crash in shard rebalancer when no distributed tables exist
+
+* Fixes a segfault caused by use after free in when using a cached connection
+
+* Fixes a UNION pushdown issue
+
+* Fixes a use after free issue that could happen when altering a distributed
+  table
+
+* Fixes an issue that could cause `citus_finish_pg_upgrade` to fail
+
+* Fixes bug about int and smallint sequences on MX
+
+* Fixes showing target shard size in the rebalance progress monitor
+
+* Fixes the relation size bug during rebalancing
+
+* Fixes two race conditions in the `get_rebalance_progress`
+
+* Improves `master_update_table_statistics` and provides distributed deadlock
```

Comment:
I think we can move these items right before `Fixes..` change log entries. These are features that feels better suited before the `fixes` items?

Reply:
I moved the fixes to the very bottom of the list. Do you think it looks good now?